haret/tests/utils/scheduler.rs

```diff
@@ -75,18 +84,24 @@ impl Scheduler {
             namespace_id: NamespaceId(Uuid::nil().to_string()),
             fsms: fsms,
             old_config: VersionedReplicas::new(),
-            new_config: new_config
+            new_config: new_config,
+            history: Vec::new()
         }
     }
 
+    pub fn log(&self) {
+        //println!("{:#?}", self.history);
```

Comment:
Either log this to a file or add a run-time switch to turn the history dump on and off during test runs.